docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc

```diff
@@ -119,9 +119,9 @@ C:\> net start CleanSpeakWebService
 
 == Maintenance Mode
 
-You will access CleanSpeak WebService’s Maintenance Mode setup via the browser. If you installed CleanSpeak WebService on your local machine, you’d access this interface by opening http://localhost:8011/ in your browser. If CleanSpeak is running on a remote server, change the server name in the URL to match your server’s name.
+You will access CleanSpeak WebService’s Maintenance Mode setup via the browser. If you installed CleanSpeak WebService on your local machine, you’d access this interface by opening `http://localhost:8011` in your browser. If CleanSpeak is running on a remote server, change the server name in the URL to match your server’s name.
```

Comment:
If we are going to use back ticks here, we should be consistent, there are a few other spots where we have the same text.